cpp/change-notes/2021-02-24-memset-may-be-deleted.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* A new query (`cpp/memset-may-be-deleted`) is run and displayed on LGTM. The query finds calls to memset that may be removed by compiler. This query was originally submitted as an experimental query by @ihsinme in https://github.com/github/codeql/pull/4953.
```

Comment:
Should a change note for `lgtm,codescanning` reference LGTM (and not codescanning) in the text?

Reply:
Yeah, maybe that's a bit weird. How about just writing "A new query (`cpp/memset-may-be-deleted`) has been added."?

Reply:
Or "A new query (`cpp/memset-may-be-deleted`) has been added.  This query is run and displayed on LGTM [and run in the codescanning suite?]."

@jbj do you have an opinion on this?  This probably won't be the last time it comes up.

Reply:
```suggestion
* A new query (`cpp/memset-may-be-deleted`) is added to the default query suite. The query finds calls to `memset` that may be removed by compiler. This query was originally [submitted as an experimental query by @ihsinme](https://github.com/github/codeql/pull/4953).
```

I've suggested an edit to side-step the question for now, and then we can have the difficult discussion when we encounter a real example of a query that works differently on Code Scanning and LGTM.